### ISSUE TITLE
MachineLearning_Toolkit: Add data preprocessing methods

### ISSUE THREAD

**Issue by BingWangUS** [BHoM/MachineLearning_Toolkit#12]
#### Description:

ref here - https://scikit-learn.org/stable/modules/classes.html#module-sklearn.preprocessing

I think the MinMaxScaler, StandardScaler and Normalizer will be useful, but can add others if someone names those.

**Comment by BingWangUS** [comment]
After discussion, adding MinMaxScaler and StandardScaler for now.
